opm/autodiff/WellInterface.hpp

```diff
@@ -198,6 +198,10 @@ namespace Opm
         virtual void addCellRates(RateVector& rates, int cellIdx) const
         {}
 
+        virtual Scalar volumetricSurfaceRateForConnection(int cellIdx, int phaseIdx) const
```

Comment:
please make this function pure virtual  (virutal func() const = 0) here, then you do not need the `{}`.

Reply:
Thanks.